### PR TITLE
Upload coverage to ScrutinizerCI from TravisCI

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
-- npm install
+  - travis_retry composer install
+  - npm install
+
 before_script:
-- phpenv config-rm xdebug.ini
-- bash build/travis/installKontoCheck.sh
-- if [ "$DB" == "mysql" ]; then bash build/travis/configureForMysql.sh; fi
-- if [ "$MCP_ACCESS_KEY" != "" ] && [ "$MCP_PROJECT_ID" != "" ]; then bash build/travis/createLocalTestConfig.sh > app/config/config.test.local.json; fi
+  - phpenv config-rm xdebug.ini
```
Comment (JeroenDeDauw) [inline .travis.yml]: Why do we do this?

**Comment by JeroenDeDauw** [comment]
Currently not working as xdebug needs to be enabled again... 

> $ if [ "$TYPE" == "coverage" ]; then phpenv config-add xdebug.ini; bash build/travis/uploadCoverage.sh; fi
File xdebug.ini does not exist.
+vendor/bin/phpunit --coverage-clover coverage.clover
PHPUnit 5.4.8 by Sebastian Bergmann and contributors.
Error:         No code coverage driver is available

**Comment by gbirke** [comment]
Looks like Scrutinizer still does not receive the coverage information.

**Comment by JeroenDeDauw** [comment]
huh? looks like it did work...

> +php ocular.phar code-coverage:upload --format=php-clover coverage.clover
#!/usr/bin/env php
Uploading code coverage for repository "g/wmde/FundraisingFrontend" and revision "cc3ed152786ed031728abf93fc79496c17f971d3"... Done

**Comment by JeroenDeDauw** [comment]
Lets see if it shows up now this is on master

**Pull request merged**